.github/workflows/clippy-checks.yml

```diff
@@ -8,8 +8,12 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
```

Comment:
I know it's not part of this PR, but... Why are we using the `nightly` toolchain here? Should we change this to `stable` 👀  cc @jmoguilevsky @guidota